### DISCUSSION TITLE
Deleting All Episodes of a Show?

### DISCUSSION THREAD

**Discussion by PeteCress** [OpenSageTV/Community#163] [General]
Used to be that if I selected a show with multiple episodes and clicked Delete, all episodes would be deleted.

But something has changed and now I have to go over to the list of episodes and delete them one at a time.

Can anybody get me back on the Good, Right, and Holy Path?

**Comment by xjim1** [comment]
You may have accidentally hit the button that switches to 'browse into group'.  If that's the case, you have to go into Detailed Setup>Customize>For Grouped Items:... and see if it says 'Select browses group'.  If it is, then toggle it.

**Comment by LoisGNS** [comment]
It depends on which UI you are using. I'm using stock Sage 7 with Malore, and if I select a group, "delete all recordings" is one of the options, along with other actions I can take on the group.

**Comment by xjim1** [comment]
I've been using Malore for so long I don't remember if there's a difference in another interface or not.  I've often accidentally hit the 'Select opens group options dialog...' button at the bottom and you no longer see the options you mention.  You have to go into Detailed setup to toggle it back.  Assuming that's the problem the OP is encountering.

**Reply by PeteCress** [reply]
Toggling "Browse Into Group" is not helping.

I am starting to wonder if, somehow, I have managed to switch to a new UI.

Somewhere along the line, I retired one of my HD200s and replaced it with an HD300 and I *think* the visual presentation on both my other HD200 and the HD300 changed to where it looks like this:
![20241013_152525](https://github.com/user-attachments/assets/70933aea-1a40-4ba8-b081-e62e2484b452)

**Comment by LoisGNS** [comment]
Try this:
Go to setup, detailed setup, customize. Around the middle of the long list is "For grouped items: Select opens options dialog, info browses into group" and make sure it is set to "select opens dialog." It's a toggle.

![image](https://github.com/user-attachments/assets/f1ac12b4-02bb-47ae-a764-3ed5b9e7ccf3)
